es-ds-components/components/es-slider.vue

```diff
@@ -47,14 +47,13 @@ const props = defineProps({
     labelFormatter: {
         type: Function,
         required: false,
-        default: (val) => val,
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
+        default: (val: any) => val,
     },
 });
 
 // allow use of v-model on this component
-const model = defineModel<number>();
-
-// set starting value
+const model = defineModel<number | number[]>();
```

Comment:
Small note, if hooked directly up to prime-vue's slider, then the value can be of type `number | number[]`.